cmd/containerboot/main.go

```diff
@@ -178,12 +179,32 @@ func main() {
 	}
 	defer killTailscaled()
 
-	if cfg.LocalAddrPort != "" && cfg.MetricsEnabled {
-		m := &metrics{
-			lc:            client,
-			debugEndpoint: cfg.DebugAddrPort,
+	var healthCheck *healthz
+	if cfg.HealthCheckAddrPort != "" {
```

Comment:
Can we deprecate and eventually remove this? We could now log a warning that it will be removed in N release (1.82) and folks should use `TS_LOCAL_ADDR_PORT` instead

Reply:
nvm, saw the warning logged later

Reply:
Well that only errors if someone tries to use old and new config together. I'll add a warning for folks who are just using the deprecated config.